src/spawn.rs

```diff
@@ -0,0 +1,78 @@
+//! Entity spawning utilities.
+
+// Unused utilities may trigger these lints undesirably.
+#![allow(dead_code)]
+
+use bevy::ecs::system::EntityCommand;
+use bevy::ecs::system::EntityCommands;
+use bevy::prelude::*;
+
+/// Re-exported extension traits.
+#[allow(unused_imports)]
+pub mod prelude {
+    pub use super::{AddExt as _, SpawnExt as _, WorldSpawnExt as _};
+}
+
+/// An extension trait that provides helper functions for deferred entity spawning.
+pub trait SpawnExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086.
+    /// Spawn an entity with an [`EntityCommand`].
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands;
+}
+
+impl SpawnExt for Commands<'_, '_> {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+impl SpawnExt for ChildBuilder<'_> {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+/// An extension trait that provides helper functions for immediate entity spawning.
+pub trait WorldSpawnExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086.
```

Comment:
```suggestion
    // Workaround for <https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086>.
```
Link does not point to the comment due to the period